packages/ui-kit/src/components/Leaderboard/Leaderboard.tsx

```diff
@@ -333,7 +333,7 @@ export const LeaderboardComponent = ({
               </td>
               {hasValueBar && (
                 <td className={ComponentStyles.valueBarCellStyles(styles)}>
-                  <ValueBar value={valuesByRow?.[rowIndex] ?? 0} maxValue={maxValue ?? 0} styles={styles} />
+                  <ValueBar value={numberValuesByRow?.[rowIndex] ?? 0} maxValue={maxValue ?? 0} styles={styles} />
```

Comment:
Instead of defaulting to `0`, should we instead just hide this if there's no numbers? Or set it to `-` like we do in the Counter case?

Reply:
Yeah, this `hasValueBar` condition is taking care of not showing the value bar if there's no numbers, as it doesn't make sense in that case. I think it's worth documenting this in the `hasValueBar` prop wdyt?

Reply:
Yes, please document 👍